### PR TITLE
eliminate shadowed variable declarations

### PR DESCRIPTION
**Pull request by craigmichaelmartin**
`col` is already defined and referencing a collection
<b>edit</b>: removed all shadowed variable declarations

### CODE REVIEW

**Comment by craigmichaelmartin** [comment]
I'd be happy to implement the eslint no-shadow if desired, unless @captbaritone has this on his radar :)

**Comment by captbaritone** [comment]
I was planning on getting to it, but don't have anything started. Go for it!

--
Jordan Eldredge

Please excuse any typos or terseness; this email was sent from a mobile device.

**Comment by captbaritone** [comment]
Lgtm

File: backbone.js [hunk 1]
```diff
@@ -893,8 +893,8 @@
       var orderChanged = false;
       var replace = !sortable && add && remove;
       if (set.length && replace) {
-        orderChanged = this.length !== set.length || _.some(this.models, function(model, index) {
-          return model !== set[index];
+        orderChanged = this.length !== set.length || _.some(this.models, function(mod, index) {
```
Comment (akre54) [inline backbone.js]: Usually our convention is just `m`.

**Comment by akre54** [comment]
Minor nit, but otherwise :+1:

**Comment by craigmichaelmartin** [comment]
Sure thing!

**Pull request merged**